devel/query.lua

```diff
@@ -274,6 +274,7 @@ function foreach(table, name, callback)
     if getmetatable(table) and table._kind and (table._kind == "enum-type" or table._kind == "bitfield-type") then
         for idx, value in ipairs(table) do
             if is_exceeding_maxlength(index) then
+                print("<query on " .. name .. " truncated after " .. index .. ">")
```

Comment:
instead of duplicating this message, could you integrate it into `is_exceeding_maxlength` (passing it the `name` param) or, if not that, make a wrapper around `is_exceeding_maxlength` that prints the message?

Reply:
"into" I don't like, cause it's an `is_*` function but I could wrap it or rename it and put it into the renamed. I'll try and do this tomorrow

Reply:
Actually a wrapper doesn't work either; needs to return when it is exceeding. I just put the print message in a nested-function.